packages/app/src/Feed/TimelineFeed.ts

```diff
@@ -147,11 +147,21 @@ export default function useTimelineFeed(subject: TimelineSubject, options: Timel
         }
         return s;
       });
-      const reposts = main.store.notes
+      const repostsByKind6 = main.store.notes
         .filter(a => a.kind === EventKind.Repost && a.content === "")
         .map(a => a.tags.find(b => b[0] === "e"))
         .filter(a => a)
         .map(a => unwrap(a)[1]);
+      const repostsByKind1 = main.store.notes
+        .filter(
+          a =>
+            (a.kind === EventKind.Repost || a.kind === EventKind.TextNote) &&
+            a.tags.some(b => b[0] === "e" && b[3] === "mention")
+        )
+        .map(a => a.tags.find((b, i) => b[0] === "e" && b[3] === "mention" && a.content === `#[${i}]`))
```

Comment:
Looks like we could extract this part of the logic to a `isTextRepost` function, seems to be used in a few places.

Reply:
fixed https://github.com/v0l/snort/commits/615c2a974ab64a4d025ea4d4f382a2da631181f9